ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -301,11 +306,19 @@ export const PageProviderDisplayBehavior = [
             }
           }
         }
-        // until we support unselect some, if the view is not handling select all, we need to prevent deselect (for now)
+        this._lastSelectedIndex = e.detail.index;
         if (this.selectAllActive && !this.handlesSelectAll) {
           this.selectItem(this.items[index]);
         }
-        this._lastSelectedIndex = e.detail.index;
+        if (this.selectAllActive && this.handlesSelectAll) {
+          if (!this._excludedItems.includes(this.items[index].uid)) {
+            this._excludedItems.push(this.items[index].uid);
+            return;
+          }
+          if (this._excludedItems.includes(this.items[index].uid)) {
```

Comment:
we can save this in variable to avoid repeat computation of `includes()` at Line 314 , 318
 `this._excludedItems.includes(this.items[index].uid)`